site/pages/api/create-consequence-network.api.ts

```diff
@@ -71,10 +77,26 @@ const createConsequenceNetwork = async (req: NextApiRequest, res: NextApiRespons
                 ? DISRUPTION_DETAIL_PAGE_PATH
                 : REVIEW_DISRUPTION_PAGE_PATH;
 
+        if (addAnotherConsequence) {
+            if (!disruption) {
+                throw new Error("No disruption found to add another consequence");
+            }
+            const nextIndex = getLargestConsequenceIndex(disruption) + 2;
```

Comment:
Index increments to 2 while adding the 0th index consequence and then clicking add another consequence - while the expected index should be 1